### ISSUE TITLE
Integrate with url parser 

### ISSUE THREAD

**Issue by chrisschaub** [alioguzhan/react-editext#109]
I would be nice to have an option to parse urls in the text and make links. So www.cnn.com would become <a href="https://www.cnn.com">https://www.cnn.com</a> on save.

**Comment by alioguzhan** [comment]
What about a more generic solution ?

We can implement a `renderView` function prop which takes the raw value as param and expects to return a modified version of the value. So with this approach, we can take the current value, parse it, alter it and return a new value (`string` or `jsx`). 

```jsx
function parseURLs(content) {
// take value here. check it. parse it.
return newContent
}

return (<EdiText value={state.text} onRenderViewMode={parseURL} />)
```

 What do you think?

**Comment by chrisschaub** [comment]
Yes, that's a great idea!

**Comment by alioguzhan** [comment]
OK great. let me think about it deeply. I will create a PR by mentioning this issue. You can follow the process.

**Comment by alioguzhan** [comment]
Hey, see below deploy preview for this feature. What do you think?

https://deploy-preview-110--react-editext.netlify.app/#render-value

**Comment by alioguzhan** [comment]
@all-contributors please add @chrisschaub  for ideas

**Comment by chrisschaub** [comment]
I'll check it out, thanks for the quick work!

**Comment by alioguzhan** [comment]
This is released under `v3.18.0`. The package can be updated via npm. Please see the examples page.
